docs/guides/writing-advanced-validations.mdx

```diff
@@ -4,6 +4,489 @@ sidebar_label: 'Writing Advanced Validations'
 
 # Writing Advanced Validations
 
-import Placeholder from '../partials/_placeholder.mdx';
+This guide provides in-depth examples for validating gameplay using [Validation Suites](../core-concepts/validation-suites).
 
-<Placeholder />
+:::note You will learn:
+✓ how to build validations for common scenarios such as navigating menus and exercising core gameplay mechanics<br/>
+✓ how to establish potential cause-and-effect relationships between actions and changes in the game state<br/>
+✓ how to build validations around non-deterministic behaviours
+:::
+
+This guide uses gameplay from [Unity's Boss Room sample project](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop).
+You can follow along by downloading Boss Room and [installing our core SDK](../getting-started/installing-regression-games),
+or you can apply the concepts covered throughout this guide to your own project.
+You can also try out our validations tools from our [Website Demo](https://play.regression.gg/validations-demo),
+although the demo lacks some gameplay from this guide.
+
+Each example describes how to recreate the gameplay it validates.
+If you're following along, see our **Core Concepts** section for instructions on
+capturing a [Gameplay Session](../core-concepts/gameplay-sessions)
+and creating a [Validation Suite](../core-concepts/validation-suites) from that Session
+to start building [Validation Scenarios](../core-concepts/validation-suites#validation-scenarios).
+
+## Changing In-Game Settings
+
+Let's start with a simple Validation Scenario.
+
+**For this Gameplay Session**: we open the settings menu then increase the music volume and toggle graphics quality from "Medium" to "High".
+
+**In the Scenario Builder**: we'll validate that the settings were changed successfully.
+
+:::note Before Recording
+Set the [recording interval](../core-concepts/recording-gameplay#adjusting-the-recording-interval) to `1`. This is necessary for capturing changes to the settings menu.
+:::
+
+![Sample Gameplay Session](img/writing-advanced-validations/changing-in-game-settings/sample-gameplay-session.gif)
```

Comment:
What tool did you end up using for the gifs? They look great!

Reply:
[ScreenToGif](https://www.screentogif.com/)
It lets you extend and remove frames, add looping, etc. Was really nice for this!